alluxio-ui/common/package.json

```diff
@@ -3,18 +3,18 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "~1.2.12",
```

Comment:
Does removing the `~` force us to always use that version?

Reply:
Yes, at the first level.